homeassistant/components/homekit_controller/__init__.py

```diff
@@ -248,7 +248,7 @@ def setup(hass, config):
     if not os.path.isdir(data_dir):
         os.mkdir(data_dir)
 
-    pairing_file = os.path.join(data_dir, 'pairings.json')
+    pairing_file = os.path.join(data_dir, 'pairing.json')
```

Comment:
Please extract the json filename string to a constant. It's used in two places.

Reply:
Done in latest commit